src/main/java/mocacong/server/service/CafeService.java

```diff
@@ -37,6 +38,7 @@
 public class CafeService {
 
     public static final int CAFE_IMAGES_PER_REQUEST_LIMIT_COUNTS = 3;
+    public static final int CAFE_IMAGES_PER_MEMBER_LIMIT_COUNTS = 3;
```

Comment:
이전에 `CAFE_IMAGES_PER_REQUEST_LIMIT_COUNTS`가 public인 상태로 머지가 돼버렸는데, private이 맞습니다. 해당 상수 역시 이곳저곳에서 쓰이기엔 응집성이 떨어질 수 있으므로 private으로 바꿔주시면 좋을 듯해요

```suggestion
    private static final int CAFE_IMAGES_PER_MEMBER_LIMIT_COUNTS = 3;
```